diff -Naur apt-2.3.6_orig/apt-pkg/contrib/srvrec.cc apt-2.3.6/apt-pkg/contrib/srvrec.cc
--- apt-2.3.6_orig/apt-pkg/contrib/srvrec.cc	2021-07-06 14:50:32.983129638 -0600
+++ apt-2.3.6/apt-pkg/contrib/srvrec.cc	2021-07-06 14:51:26.515585419 -0600
@@ -71,7 +71,7 @@
    // Close on return
    std::shared_ptr<void> guard(&res, res_nclose);
 
-   answer_len = res_nquery(&res, name.c_str(), C_IN, T_SRV, answer, sizeof(answer));
+   answer_len = res_query(name.c_str(), C_IN, T_SRV, answer, sizeof(answer));
 #else
    if (res_init() != 0)
       return _error->Errno("res_init", "Failed to init resolver");
