diff -Naur apt-2.5.2.orig/apt-pkg/contrib/fileutl.h apt-2.5.2/apt-pkg/contrib/fileutl.h
--- apt-2.5.2.orig/apt-pkg/contrib/fileutl.h	2022-08-15 08:37:18.948538057 -0600
+++ apt-2.5.2/apt-pkg/contrib/fileutl.h	2022-08-15 08:39:45.280536990 -0600
@@ -27,6 +27,7 @@
 #include <string>
 #include <vector>
 #include <time.h>
+#include <sys/stat.h>
 
 /* Define this for python-apt */
 #define APT_HAS_GZIP 1
