diff -Naur asterisk-chan-dongle.orig/configure.ac asterisk-chan-dongle/configure.ac
--- asterisk-chan-dongle.orig/configure.ac	2017-12-29 12:45:31.679192831 -0700
+++ asterisk-chan-dongle/configure.ac	2017-12-29 12:48:36.657061546 -0700
@@ -21,7 +21,7 @@
 	[asterisk],
 	AS_HELP_STRING([--with-asterisk=path], [set asterisk headers location]),
 	[ if test "x$with_asterisk" = "xyes" -o "x$with_asterisk" = "xno" ; then  AC_MSG_ERROR([Invalid --with-asterisk=path value]); fi ],
-	[ with_asterisk="../include /usr/include /usr/local/include /opt/local/include" ]
+	[ with_asterisk="$DEST/include" ]
 )
 
 dnl  Required: asterisk version
@@ -102,7 +102,7 @@
 )
 
 AC_HEADER_FIND([asterisk.h], $with_asterisk)
-AC_HEADER_FIND([iconv.h], /usr/include /usr/local/include /opt/local/include)
+AC_HEADER_FIND([iconv.h], $DEST/include)
 
 AC_DEFINE([ICONV_CONST],[], [Define to const if you has iconv() const declaration of input buffer])
 AC_MSG_CHECKING([for iconv use const inbuf])
@@ -200,7 +200,7 @@
     fi
     ])
 
-AC_CHECK_DESTDIR([/usr/lib/asterisk/modules /usr/local/lib/asterisk/modules /opt/local/lib/asterisk/modules])
+AC_CHECK_DESTDIR([$DEST/lib/asterisk/modules])
 
 dnl Checks for library functions.
 AC_FUNC_MEMCMP
@@ -212,7 +212,7 @@
   CFLAGS="$CFLAGS -O0 -g"
   AC_DEFINE([__DEBUG__], [1], [Build with debugging])
 else
-  CFLAGS="$CFLAGS -O6"
+  CFLAGS="$CFLAGS -O2"
 fi
 
 dnl Asterisk header files use lots of old style declarations, ignore those.
@@ -230,7 +230,7 @@
 
 case "$target_os" in
     linux*)
-	SOLINK="-shared -Xlinker -x"
+	SOLINK="-shared"
 	DC_LDFLAGS=""
 	;;
     freebsd*)
