*** /home/lance/mt-sde	2012-10-03 19:04:29.140596001 -0600
--- ./mt-sde	2012-10-03 19:05:01.424596001 -0600
***************
*** 6,10 ****
  # has two purposes: it allows libraries to be used in situations where
  # $gp != our _gp, and it allows them to be built with -G8 while
  # retaining link compability with -G0 and -G4.
! CFLAGS_FOR_TARGET += -Os -minterlink-mips16 -mcode-xonly -mno-gpopt
  CXXFLAGS_FOR_TARGET += -Os -minterlink-mips16 -mcode-xonly -mno-gpopt
--- 6,10 ----
  # has two purposes: it allows libraries to be used in situations where
  # $gp != our _gp, and it allows them to be built with -G8 while
  # retaining link compability with -G0 and -G4.
! CFLAGS_FOR_TARGET += -Os -mcode-xonly -mno-gpopt
  CXXFLAGS_FOR_TARGET += -Os -minterlink-mips16 -mcode-xonly -mno-gpopt
