--- makefile.orig	2014-03-01 02:42:42.525022145 -0700
+++ makefile	2014-03-01 02:44:22.224241380 -0700
@@ -2,12 +2,12 @@
 # Makefile for UNIX - unrar
 
 # Linux using GCC
-CXX=g++
-CXXFLAGS=-O2
+CXX=mipsel-linux-g++
+CXXFLAGS=-O2 -mtune=mips32 -mips32
 LIBFLAGS=-fPIC
 DEFINES=-D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -DRAR_SMP
-STRIP=strip
-LDFLAGS=-pthread
+STRIP=mipsel-linux-strip
+LDFLAGS=-pthread -Wl,--dynamic-linker=$(PREFIX)/lib/ld-uClibc.so.0 -Wl,-rpath,$(PREFIX)/lib
 DESTDIR=/usr
 
 # Linux using LCC
