--- unix/Makefile.orig	2014-03-01 12:37:11.026641894 -0700
+++ unix/Makefile	2014-03-01 12:40:57.762113332 -0700
@@ -50,7 +50,7 @@
 CFLAGS = -O
 CF_NOOPT = -I. -I$(IZ_BZIP2) -DUNIX $(LOC)
 CF = $(CFLAGS) $(CF_NOOPT)
-LFLAGS1 =
+LFLAGS1 = -Wl,--dynamic-linker=$(PREFIX)/lib/ld-uClibc.so.0 -Wl,-rpath,$(RPATH)
 LF = -o unzip$E $(LFLAGS1)
 LF2 = -s
 
@@ -819,7 +819,7 @@
 # Linux (Posix, approximately SysV):  virtually any version since before 0.96,
 # for any platform.  Change "-O" to "-O3" or whatever, as desired...
 linux_noasm:	unix_make
-	$(MAKE) unzips CC=gcc LD=gcc CFLAGS="-O -Wall"
+	$(MAKE) unzips CC=mipsel-linux-gcc LD=mipsel-linux-gcc CFLAGS="-O -mtune=mips32 -mips32 -Wall -DNO_LCHMOD"
 
 # Linux with lcc compiler:  __inline__ (stat.h) not recognized, and must edit
 # /usr/include/gnu/types.h to get rid of "long long" if __LCC__ defined.  -O3
