diff -Naur src_orig/crypto/x509/root_linux.go src/crypto/x509/root_linux.go
--- src_orig/crypto/x509/root_linux.go	2020-05-29 08:58:56.120667993 -0600
+++ src/crypto/x509/root_linux.go	2020-05-29 09:55:03.594284180 -0600
@@ -6,6 +6,8 @@
 
 // Possible certificate files; stop after finding one.
 var certFiles = []string{
+	"PREFIX/ssl/certs/ca-certificates.crt",              // Tomatoware
+	"/rom/cacert.pem",                                   // Tomato
 	"/etc/ssl/certs/ca-certificates.crt",                // Debian/Ubuntu/Gentoo etc.
 	"/etc/pki/tls/certs/ca-bundle.crt",                  // Fedora/RHEL 6
 	"/etc/ssl/ca-bundle.pem",                            // OpenSUSE
