diff -Naur gcc-10.1.0_orig/libgo/configure gcc-10.1.0/libgo/configure
--- gcc-10.1.0_orig/libgo/configure	2020-06-29 10:40:43.876021398 -0600
+++ gcc-10.1.0/libgo/configure	2020-06-29 10:41:19.168366983 -0600
@@ -14090,7 +14090,7 @@
     GOARCH=arm
     case ${host} in
       arm*b*-*-*)
-	GOARCH=armbe
+	GOARCH=arm
 	;;
     esac
     ;;
diff -Naur gcc-10.1.0_orig/libgo/configure.ac gcc-10.1.0/libgo/configure.ac
--- gcc-10.1.0_orig/libgo/configure.ac	2020-06-29 10:40:43.876021398 -0600
+++ gcc-10.1.0/libgo/configure.ac	2020-06-29 10:41:19.168366983 -0600
@@ -256,7 +256,7 @@
     GOARCH=arm
     case ${host} in
       arm*b*-*-*)
-	GOARCH=armbe
+	GOARCH=arm
 	;;
     esac
     ;;
