diff -Naur gcc-11.2.0_orig/libgo/configure gcc-11.2.0/libgo/configure
--- gcc-11.2.0_orig/libgo/configure	2021-08-23 08:24:14.063539409 -0600
+++ gcc-11.2.0/libgo/configure	2021-08-23 08:44:15.317493295 -0600
@@ -14144,7 +14144,7 @@
     GOARCH=arm
     case ${host} in
       arm*b*-*-*)
-	GOARCH=armbe
+	GOARCH=arm
 	;;
     esac
     ;;
diff -Naur gcc-11.2.0_orig/libgo/configure.ac gcc-11.2.0/libgo/configure.ac
--- gcc-11.2.0_orig/libgo/configure.ac	2021-08-23 08:24:14.063539409 -0600
+++ gcc-11.2.0/libgo/configure.ac	2021-08-23 08:44:15.317493295 -0600
@@ -260,7 +260,7 @@
     GOARCH=arm
     case ${host} in
       arm*b*-*-*)
-	GOARCH=armbe
+	GOARCH=arm
 	;;
     esac
     ;;
