diff -Naur gcc-11.2.0_orig/libgo/mkruntimeinc.sh gcc-11.2.0/libgo/mkruntimeinc.sh
--- gcc-11.2.0_orig/libgo/mkruntimeinc.sh	2021-08-23 08:24:14.063539409 -0600
+++ gcc-11.2.0/libgo/mkruntimeinc.sh	2021-08-23 08:37:57.741102636 -0600
@@ -24,7 +24,7 @@
 do
   grep "#define $pattern" ${IN} >> runtime.inc.tmp2
 done
-TYPES="_Complex_lock _Reader_lock semt boundsError"
+TYPES="_Complex_lock _Reader_lock semt boundsError _FILE"
 for TYPE in $TYPES
 do
   sed -e '/struct '${TYPE}' {/,/^}/s/^.*$//' runtime.inc.tmp2 > runtime.inc.tmp3;
