diff -Naur gcc-11.2.0_orig/libgo/go/syscall/syscall_linux_mipsx.go gcc-11.2.0/libgo/go/syscall/syscall_linux_mipsx.go
--- gcc-11.2.0_orig/libgo/go/syscall/syscall_linux_mipsx.go	2021-08-23 17:21:23.869462530 -0600
+++ gcc-11.2.0/libgo/go/syscall/syscall_linux_mipsx.go	2021-08-23 21:52:21.457823114 -0600
@@ -9,11 +9,11 @@
 
 import "unsafe"
 
-func (r *PtraceRegs) PC() uint64 {
+func (r *PtraceRegs) PC() uint32 {
 	return r.Cp0_epc
 }
 
-func (r *PtraceRegs) SetPC(pc uint64) {
+func (r *PtraceRegs) SetPC(pc uint32) {
 	r.Cp0_epc = pc
 }
 
