diff -Naur gcc-12.1.0_orig/libcc1/libcc1plugin.cc gcc-12.1.0/libcc1/libcc1plugin.cc
--- gcc-12.1.0_orig/libcc1/libcc1plugin.cc	2022-05-17 10:37:53.777943722 -0600
+++ gcc-12.1.0/libcc1/libcc1plugin.cc	2022-05-17 11:37:51.502140187 -0600
@@ -31,6 +31,7 @@
 #undef PACKAGE_TARNAME
 #undef PACKAGE_VERSION
 
+#define INCLUDE_MEMORY
 #include "gcc-plugin.h"
 #include "system.h"
 #include "coretypes.h"
diff -Naur gcc-12.1.0_orig/libcc1/libcp1plugin.cc gcc-12.1.0/libcc1/libcp1plugin.cc
--- gcc-12.1.0_orig/libcc1/libcp1plugin.cc	2022-05-17 10:37:53.777943722 -0600
+++ gcc-12.1.0/libcc1/libcp1plugin.cc	2022-05-17 11:37:51.502140187 -0600
@@ -32,6 +32,7 @@
 #undef PACKAGE_TARNAME
 #undef PACKAGE_VERSION
 
+#define INCLUDE_MEMORY
 #include "gcc-plugin.h"
 #include "system.h"
 #include "coretypes.h"
