diff -Naur gcc-12.2.0.orig/gcc/config/aarch64/aarch64-linux.h gcc-12.2.0/gcc/config/aarch64/aarch64-linux.h
--- gcc-12.2.0.orig/gcc/config/aarch64/aarch64-linux.h	2022-08-19 02:09:52.552662114 -0600
+++ gcc-12.2.0/gcc/config/aarch64/aarch64-linux.h	2022-10-17 07:34:09.719361583 -0600
@@ -24,7 +24,7 @@
 #define GLIBC_DYNAMIC_LINKER "/lib/ld-linux-aarch64%{mbig-endian:_be}%{mabi=ilp32:_ilp32}.so.1"
 
 #undef MUSL_DYNAMIC_LINKER
-#define MUSL_DYNAMIC_LINKER "/lib/ld-musl-aarch64%{mbig-endian:_be}%{mabi=ilp32:_ilp32}.so.1"
+#define MUSL_DYNAMIC_LINKER "/opt/lib/ld-musl-aarch64%{mbig-endian:_be}%{mabi=ilp32:_ilp32}.so.1"
 
 #undef  ASAN_CC1_SPEC
 #define ASAN_CC1_SPEC "%{%:sanitize(address):-funwind-tables}"
diff -Naur gcc-12.2.0.orig/gcc/config/arm/linux-eabi.h gcc-12.2.0/gcc/config/arm/linux-eabi.h
--- gcc-12.2.0.orig/gcc/config/arm/linux-eabi.h	2022-08-19 02:09:52.584662523 -0600
+++ gcc-12.2.0/gcc/config/arm/linux-eabi.h	2022-10-17 07:34:09.723361575 -0600
@@ -89,7 +89,7 @@
 #define MUSL_DYNAMIC_LINKER_E "%{mbig-endian:eb}"
 #endif
 #define MUSL_DYNAMIC_LINKER \
-  "/lib/ld-musl-arm" MUSL_DYNAMIC_LINKER_E "%{mfloat-abi=hard:hf}%{mfdpic:-fdpic}.so.1"
+  "/opt/lib/ld-musl-arm" MUSL_DYNAMIC_LINKER_E "%{mfloat-abi=hard:hf}%{mfdpic:-fdpic}.so.1"
 
 /* At this point, bpabi.h will have clobbered LINK_SPEC.  We want to
    use the GNU/Linux version, not the generic BPABI version.  */
diff -Naur gcc-12.2.0.orig/gcc/config/i386/linux64.h gcc-12.2.0/gcc/config/i386/linux64.h
--- gcc-12.2.0.orig/gcc/config/i386/linux64.h	2022-08-19 02:09:52.632663137 -0600
+++ gcc-12.2.0/gcc/config/i386/linux64.h	2022-10-17 07:34:30.683318733 -0600
@@ -32,8 +32,8 @@
 #define GLIBC_DYNAMIC_LINKERX32 "/libx32/ld-linux-x32.so.2"
 
 #undef MUSL_DYNAMIC_LINKER32
-#define MUSL_DYNAMIC_LINKER32 "/lib/ld-musl-i386.so.1"
+#define MUSL_DYNAMIC_LINKER32 "/opt/lib/ld-musl-i386.so.1"
 #undef MUSL_DYNAMIC_LINKER64
-#define MUSL_DYNAMIC_LINKER64 "/lib/ld-musl-x86_64.so.1"
+#define MUSL_DYNAMIC_LINKER64 "/opt/lib/ld-musl-x86_64.so.1"
 #undef MUSL_DYNAMIC_LINKERX32
-#define MUSL_DYNAMIC_LINKERX32 "/lib/ld-musl-x32.so.1"
+#define MUSL_DYNAMIC_LINKERX32 "/opt/lib/ld-musl-x32.so.1"
diff -Naur gcc-12.2.0.orig/gcc/config/linux.h gcc-12.2.0/gcc/config/linux.h
--- gcc-12.2.0.orig/gcc/config/linux.h	2022-08-19 02:09:52.648663341 -0600
+++ gcc-12.2.0/gcc/config/linux.h	2022-10-17 07:34:09.723361575 -0600
@@ -99,10 +99,10 @@
    GLIBC_DYNAMIC_LINKER must be defined for each target using them, or
    GLIBC_DYNAMIC_LINKER32 and GLIBC_DYNAMIC_LINKER64 for targets
    supporting both 32-bit and 64-bit compilation.  */
-#define UCLIBC_DYNAMIC_LINKER "/lib/ld-uClibc.so.0"
-#define UCLIBC_DYNAMIC_LINKER32 "/lib/ld-uClibc.so.0"
-#define UCLIBC_DYNAMIC_LINKER64 "/lib/ld64-uClibc.so.0"
-#define UCLIBC_DYNAMIC_LINKERX32 "/lib/ldx32-uClibc.so.0"
+#define UCLIBC_DYNAMIC_LINKER "/opt/lib/ld-uClibc.so.1"
+#define UCLIBC_DYNAMIC_LINKER32 "/opt/lib/ld-uClibc.so.1"
+#define UCLIBC_DYNAMIC_LINKER64 "/opt/lib/ld64-uClibc.so.1"
+#define UCLIBC_DYNAMIC_LINKERX32 "/opt/lib/ldx32-uClibc.so.1"
 #define BIONIC_DYNAMIC_LINKER "/system/bin/linker"
 #define BIONIC_DYNAMIC_LINKER32 "/system/bin/linker"
 #define BIONIC_DYNAMIC_LINKER64 "/system/bin/linker64"
diff -Naur gcc-12.2.0.orig/gcc/config/mips/linux.h gcc-12.2.0/gcc/config/mips/linux.h
--- gcc-12.2.0.orig/gcc/config/mips/linux.h	2022-08-19 02:09:52.660663495 -0600
+++ gcc-12.2.0/gcc/config/mips/linux.h	2022-10-17 07:34:09.723361575 -0600
@@ -30,12 +30,12 @@
 
 #undef UCLIBC_DYNAMIC_LINKER32
 #define UCLIBC_DYNAMIC_LINKER32 \
-  "%{mnan=2008:/lib/ld-uClibc-mipsn8.so.0;:/lib/ld-uClibc.so.0}"
+  "%{mnan=2008:/opt/lib/ld-uClibc-mipsn8.so.1;:/opt/lib/ld-uClibc.so.1}"
 #undef UCLIBC_DYNAMIC_LINKER64
 #define UCLIBC_DYNAMIC_LINKER64 \
-  "%{mnan=2008:/lib/ld64-uClibc-mipsn8.so.0;:/lib/ld64-uClibc.so.0}"
+  "%{mnan=2008:/opt/lib/ld64-uClibc-mipsn8.so.1;:/opt/lib/ld64-uClibc.so.1}"
 #define UCLIBC_DYNAMIC_LINKERN32 \
-  "%{mnan=2008:/lib32/ld-uClibc-mipsn8.so.0;:/lib32/ld-uClibc.so.0}"
+  "%{mnan=2008:/opt/lib32/ld-uClibc-mipsn8.so.1;:/opt/lib32/ld-uClibc.so.1}"
 
 #undef MUSL_DYNAMIC_LINKER32
 #define MUSL_DYNAMIC_LINKER32 \
