diff -Naur dpkg-1.21.2_orig/src/main/remove.c dpkg-1.21.2/src/main/remove.c
--- dpkg-1.21.2_orig/src/main/remove.c	2022-03-21 08:19:42.716206059 -0600
+++ dpkg-1.21.2/src/main/remove.c	2022-03-21 08:19:57.456329340 -0600
@@ -448,8 +448,8 @@
     debug(dbg_eachfiledetail, "removal_bulk removing '%s'", fnvb.buf);
     if (!rmdir(fnvb.buf) || errno == ENOENT || errno == ELOOP) continue;
     if (errno == ENOTEMPTY || errno == EEXIST) {
-      warning(_("while removing %.250s, directory '%.250s' not empty so not removed"),
-              pkg_name(pkg, pnaw_nonambig), namenode->name);
+//      warning(_("while removing %.250s, directory '%.250s' not empty so not removed"),
+//              pkg_name(pkg, pnaw_nonambig), namenode->name);
       push_leftover(&leftover,namenode);
       continue;
     } else if (errno == EBUSY || errno == EPERM) {
