diff -Naur llvm-project-12.0.0.orig/lldb/source/Host/posix/ProcessLauncherPosixFork.cpp llvm-project-12.0.0/lldb/source/Host/posix/ProcessLauncherPosixFork.cpp
--- llvm-project-12.0.0.orig/lldb/source/Host/posix/ProcessLauncherPosixFork.cpp	2021-04-23 11:02:43.539763330 -0600
+++ llvm-project-12.0.0/lldb/source/Host/posix/ProcessLauncherPosixFork.cpp	2021-04-23 11:03:12.571965347 -0600
@@ -63,7 +63,7 @@
     if (value == -1)
       ExitWithError(error_fd, "personality get");
 
-    value = personality(ADDR_NO_RANDOMIZE | value);
+    value = personality(0x0040000 | value);
     if (value == -1)
       ExitWithError(error_fd, "personality set");
   }
