diff -Naur llvm-project-14.0.0.src.orig/clang/lib/Driver/ToolChains/Arch/ARM.cpp llvm-project-14.0.0.src/clang/lib/Driver/ToolChains/Arch/ARM.cpp
--- llvm-project-14.0.0.src.orig/clang/lib/Driver/ToolChains/Arch/ARM.cpp	2022-03-14 03:44:55.000000000 -0600
+++ llvm-project-14.0.0.src/clang/lib/Driver/ToolChains/Arch/ARM.cpp	2022-03-25 13:20:41.507246814 -0600
@@ -370,7 +370,7 @@
     case llvm::Triple::MuslEABI:
     case llvm::Triple::EABI:
       // EABI is always AAPCS, and if it was not marked 'hard', it's softfp
-      return FloatABI::SoftFP;
+      return FloatABI::Soft;
     case llvm::Triple::Android:
       return (SubArch >= 7) ? FloatABI::SoftFP : FloatABI::Soft;
     default:
