diff -Naur llvm-project-12.0.0.orig/clang/lib/Driver/ToolChains/Arch/ARM.cpp llvm-project-12.0.0/clang/lib/Driver/ToolChains/Arch/ARM.cpp
--- llvm-project-12.0.0.orig/clang/lib/Driver/ToolChains/Arch/ARM.cpp	2021-04-23 11:05:33.328947576 -0600
+++ llvm-project-12.0.0/clang/lib/Driver/ToolChains/Arch/ARM.cpp	2021-04-23 11:08:42.286271945 -0600
@@ -220,7 +220,7 @@
     case llvm::Triple::MuslEABI:
     case llvm::Triple::EABI:
       // EABI is always AAPCS, and if it was not marked 'hard', it's softfp
-      return FloatABI::SoftFP;
+      return FloatABI::Soft;
     case llvm::Triple::Android:
       return (SubArch >= 7) ? FloatABI::SoftFP : FloatABI::Soft;
     default:
