diff -Naur llvm-project-12.0.0.orig/clang/lib/Driver/ToolChains/Darwin.cpp llvm-project-12.0.0/clang/lib/Driver/ToolChains/Darwin.cpp
--- llvm-project-12.0.0.orig/clang/lib/Driver/ToolChains/Darwin.cpp	2021-04-23 11:05:33.332947604 -0600
+++ llvm-project-12.0.0/clang/lib/Driver/ToolChains/Darwin.cpp	2021-04-23 11:10:09.674886146 -0600
@@ -1981,9 +1981,12 @@
     llvm::SmallVector<llvm::StringRef, 5> dirs;
     CIncludeDirs.split(dirs, ":");
     for (llvm::StringRef dir : dirs) {
-      llvm::StringRef Prefix =
-          llvm::sys::path::is_absolute(dir) ? "" : llvm::StringRef(Sysroot);
-      addExternCSystemInclude(DriverArgs, CC1Args, Prefix + dir);
+      if (llvm::sys::path::is_absolute(dir))
+        addExternCSystemInclude(DriverArgs, CC1Args, dir);
+      else {
+        addExternCSystemInclude(DriverArgs, CC1Args,
+                                StringRef(Sysroot) + "/" + dir);
+      }
     }
   } else {
     // Otherwise, add <sysroot>/usr/include.
diff -Naur llvm-project-12.0.0.orig/clang/lib/Driver/ToolChains/Fuchsia.cpp llvm-project-12.0.0/clang/lib/Driver/ToolChains/Fuchsia.cpp
--- llvm-project-12.0.0.orig/clang/lib/Driver/ToolChains/Fuchsia.cpp	2021-04-23 11:05:33.328947576 -0600
+++ llvm-project-12.0.0/clang/lib/Driver/ToolChains/Fuchsia.cpp	2021-04-23 11:10:09.674886146 -0600
@@ -320,9 +320,12 @@
     SmallVector<StringRef, 5> dirs;
     CIncludeDirs.split(dirs, ":");
     for (StringRef dir : dirs) {
-      StringRef Prefix =
-          llvm::sys::path::is_absolute(dir) ? "" : StringRef(D.SysRoot);
-      addExternCSystemInclude(DriverArgs, CC1Args, Prefix + dir);
+      if (llvm::sys::path::is_absolute(dir))
+        addExternCSystemInclude(DriverArgs, CC1Args, dir);
+      else {
+        addExternCSystemInclude(DriverArgs, CC1Args,
+                                StringRef(D.SysRoot) + "/" + dir);
+      }
     }
     return;
   }
diff -Naur llvm-project-12.0.0.orig/clang/lib/Driver/ToolChains/Hurd.cpp llvm-project-12.0.0/clang/lib/Driver/ToolChains/Hurd.cpp
--- llvm-project-12.0.0.orig/clang/lib/Driver/ToolChains/Hurd.cpp	2021-04-23 11:05:33.328947576 -0600
+++ llvm-project-12.0.0/clang/lib/Driver/ToolChains/Hurd.cpp	2021-04-23 11:10:09.674886146 -0600
@@ -158,9 +158,12 @@
     SmallVector<StringRef, 5> Dirs;
     CIncludeDirs.split(Dirs, ":");
     for (StringRef Dir : Dirs) {
-      StringRef Prefix =
-          llvm::sys::path::is_absolute(Dir) ? "" : StringRef(SysRoot);
-      addExternCSystemInclude(DriverArgs, CC1Args, Prefix + Dir);
+      if (llvm::sys::path::is_absolute(Dir))
+        addExternCSystemInclude(DriverArgs, CC1Args, Dir);
+      else {
+        addExternCSystemInclude(DriverArgs, CC1Args,
+                                StringRef(SysRoot) + "/" + Dir);
+      }
     }
     return;
   }
diff -Naur llvm-project-12.0.0.orig/clang/lib/Driver/ToolChains/Linux.cpp llvm-project-12.0.0/clang/lib/Driver/ToolChains/Linux.cpp
--- llvm-project-12.0.0.orig/clang/lib/Driver/ToolChains/Linux.cpp	2021-04-23 11:05:33.328947576 -0600
+++ llvm-project-12.0.0/clang/lib/Driver/ToolChains/Linux.cpp	2021-04-23 11:10:09.674886146 -0600
@@ -586,9 +586,12 @@
     SmallVector<StringRef, 5> dirs;
     CIncludeDirs.split(dirs, ":");
     for (StringRef dir : dirs) {
-      StringRef Prefix =
-          llvm::sys::path::is_absolute(dir) ? "" : StringRef(SysRoot);
-      addExternCSystemInclude(DriverArgs, CC1Args, Prefix + dir);
+      if (llvm::sys::path::is_absolute(dir))
+        addExternCSystemInclude(DriverArgs, CC1Args, dir);
+      else {
+        addExternCSystemInclude(DriverArgs, CC1Args,
+                                StringRef(SysRoot) + "/" + dir);
+      }
     }
     return;
   }
diff -Naur llvm-project-12.0.0.orig/clang/lib/Driver/ToolChains/OpenBSD.cpp llvm-project-12.0.0/clang/lib/Driver/ToolChains/OpenBSD.cpp
--- llvm-project-12.0.0.orig/clang/lib/Driver/ToolChains/OpenBSD.cpp	2021-04-23 11:05:33.328947576 -0600
+++ llvm-project-12.0.0/clang/lib/Driver/ToolChains/OpenBSD.cpp	2021-04-23 11:10:09.674886146 -0600
@@ -274,9 +274,12 @@
     SmallVector<StringRef, 5> dirs;
     CIncludeDirs.split(dirs, ":");
     for (StringRef dir : dirs) {
-      StringRef Prefix =
-          llvm::sys::path::is_absolute(dir) ? StringRef(D.SysRoot) : "";
-      addExternCSystemInclude(DriverArgs, CC1Args, Prefix + dir);
+      if (llvm::sys::path::is_absolute(dir))
+        addExternCSystemInclude(DriverArgs, CC1Args, dir);
+      else {
+        addExternCSystemInclude(DriverArgs, CC1Args,
+                                StringRef(D.SysRoot) + "/" + dir);
+      }
     }
     return;
   }
diff -Naur llvm-project-12.0.0.orig/clang/lib/Driver/ToolChains/Solaris.cpp llvm-project-12.0.0/clang/lib/Driver/ToolChains/Solaris.cpp
--- llvm-project-12.0.0.orig/clang/lib/Driver/ToolChains/Solaris.cpp	2021-04-23 11:05:33.328947576 -0600
+++ llvm-project-12.0.0/clang/lib/Driver/ToolChains/Solaris.cpp	2021-04-23 11:10:09.674886146 -0600
@@ -245,9 +245,12 @@
     SmallVector<StringRef, 5> dirs;
     CIncludeDirs.split(dirs, ":");
     for (StringRef dir : dirs) {
-      StringRef Prefix =
-          llvm::sys::path::is_absolute(dir) ? "" : StringRef(D.SysRoot);
-      addExternCSystemInclude(DriverArgs, CC1Args, Prefix + dir);
+      if (llvm::sys::path::is_absolute(dir))
+        addExternCSystemInclude(DriverArgs, CC1Args, dir);
+      else {
+        addExternCSystemInclude(DriverArgs, CC1Args,
+                                StringRef(D.SysRoot) + "/" + dir);
+      }
     }
     return;
   }
diff -Naur llvm-project-12.0.0.orig/clang/lib/Driver/ToolChains/WebAssembly.cpp llvm-project-12.0.0/clang/lib/Driver/ToolChains/WebAssembly.cpp
--- llvm-project-12.0.0.orig/clang/lib/Driver/ToolChains/WebAssembly.cpp	2021-04-23 11:05:33.328947576 -0600
+++ llvm-project-12.0.0/clang/lib/Driver/ToolChains/WebAssembly.cpp	2021-04-23 11:10:09.674886146 -0600
@@ -331,9 +331,12 @@
     SmallVector<StringRef, 5> dirs;
     CIncludeDirs.split(dirs, ":");
     for (StringRef dir : dirs) {
-      StringRef Prefix =
-          llvm::sys::path::is_absolute(dir) ? "" : StringRef(D.SysRoot);
-      addExternCSystemInclude(DriverArgs, CC1Args, Prefix + dir);
+      if (llvm::sys::path::is_absolute(dir))
+        addExternCSystemInclude(DriverArgs, CC1Args, dir);
+      else {
+        addExternCSystemInclude(DriverArgs, CC1Args,
+                                StringRef(D.SysRoot) + "/" + dir);
+      }
     }
     return;
   }
