diff -Naur llvm-project-14.0.0.src.orig/third-party/benchmark/src/sysinfo.cc llvm-project-14.0.0.src/third-party/benchmark/src/sysinfo.cc
--- llvm-project-14.0.0.src.orig/third-party/benchmark/src/sysinfo.cc	2022-03-14 03:44:55.000000000 -0600
+++ llvm-project-14.0.0.src/third-party/benchmark/src/sysinfo.cc	2022-03-31 13:34:43.581146526 -0600
@@ -687,7 +687,7 @@
 #if (defined BENCHMARK_OS_FREEBSD || defined(BENCHMARK_OS_LINUX) ||     \
      defined BENCHMARK_OS_MACOSX || defined BENCHMARK_OS_NETBSD ||      \
      defined BENCHMARK_OS_OPENBSD || defined BENCHMARK_OS_DRAGONFLY) && \
-    !defined(__ANDROID__)
+    !defined(__ANDROID__) && !defined(__UCLIBC__)
   constexpr int kMaxSamples = 3;
   std::vector<double> res(kMaxSamples, 0.0);
   const int nelem = getloadavg(res.data(), kMaxSamples);
