diff -Naur llvm-project.orig/clang/lib/Driver/ToolChains/Linux.cpp llvm-project/clang/lib/Driver/ToolChains/Linux.cpp
--- llvm-project.orig/clang/lib/Driver/ToolChains/Linux.cpp	2019-04-09 10:01:01.261123030 -0600
+++ llvm-project/clang/lib/Driver/ToolChains/Linux.cpp	2019-04-09 10:02:06.985625049 -0600
@@ -562,8 +562,8 @@
         Triple.getEnvironment() == llvm::Triple::GNUEABIHF ||
         tools::arm::getARMFloatABI(*this, Args) == tools::arm::FloatABI::Hard;
 
-    LibDir = "lib";
-    Loader = HF ? "ld-linux-armhf.so.3" : "ld-linux.so.3";
+    LibDir = "mmc/lib";
+    Loader = HF ? "ld-uClibc.so.1" : "ld-uClibc.so.1";
     break;
   }
   case llvm::Triple::mips:
@@ -572,16 +572,16 @@
   case llvm::Triple::mips64el: {
     bool IsNaN2008 = tools::mips::isNaN2008(Args, Triple);
 
-    LibDir = "lib" + tools::mips::getMipsABILibSuffix(Args, Triple);
+    LibDir = "mmc/lib" + tools::mips::getMipsABILibSuffix(Args, Triple);
 
     if (tools::mips::isUCLibc(Args))
-      Loader = IsNaN2008 ? "ld-uClibc-mipsn8.so.0" : "ld-uClibc.so.0";
+      Loader = IsNaN2008 ? "ld-uClibc.so.1" : "ld-uClibc.so.1";
     else if (!Triple.hasEnvironment() &&
              Triple.getVendor() == llvm::Triple::VendorType::MipsTechnologies)
       Loader =
           Triple.isLittleEndian() ? "ld-musl-mipsel.so.1" : "ld-musl-mips.so.1";
     else
-      Loader = IsNaN2008 ? "ld-linux-mipsn8.so.1" : "ld.so.1";
+      Loader = IsNaN2008 ? "ld-uClibc.so.1" : "ld-uClibc.so.1";
 
     break;
   }
