diff -Nuar unrar.orig/makefile unrar/makefile
--- unrar.orig/makefile	2016-06-15 14:18:34.082996311 -0600
+++ unrar/makefile	2016-06-15 14:19:38.523448758 -0600
@@ -2,12 +2,12 @@
 # Makefile for UNIX - unrar
 
 # Linux using GCC
-CXX=c++
-CXXFLAGS=-O2
+CXX=$(DESTARCH)-linux-g++
+CXXFLAGS=-O2 $(EXTRACFLAGS)
 LIBFLAGS=-fPIC
 DEFINES=-D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -DRAR_SMP
-STRIP=strip
-LDFLAGS=-pthread
+STRIP=$(DESTARCH)-linux-strip
+LDFLAGS=-pthread -Wl,--dynamic-linker=$(PREFIX)/lib/ld-uClibc.so.1 -Wl,-rpath,$(PREFIX)/lib
 DESTDIR=/usr
 
 # Linux using LCC
