diff -Naur unrar.orig/makefile unrar/makefile
--- unrar.orig/makefile	2024-07-10 10:17:55.491715893 -0600
+++ unrar/makefile	2024-07-10 10:18:49.079781315 -0600
@@ -2,13 +2,13 @@
 # Makefile for UNIX - unrar
 
 # Linux using GCC
-CXX=c++
-CXXFLAGS=-march=native -O2 -std=c++11 -Wno-logical-op-parentheses -Wno-switch -Wno-dangling-else
+CXX=$(DESTARCH)-tomatoware-linux-$(DESTARCHLIBC)$(EABI)-g++
+CXXFLAGS=-O2 -Wno-logical-op-parentheses -Wno-switch -Wno-dangling-else $(EXTRACFLAGS)
 LIBFLAGS=-fPIC
 DEFINES=-D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -DRAR_SMP
-STRIP=strip
-AR=ar
-LDFLAGS=-pthread
+STRIP=$(DESTARCH)-tomatoware-linux-$(DESTARCHLIBC)$(EABI)-strip
+AR=$(DESTARCH)-tomatoware-linux-$(DESTARCHLIBC)$(EABI)-ar
+LDFLAGS=$(_LDFLAGS)
 DESTDIR=/usr
 
 ##########################
