diff -Naur unrar.orig/makefile unrar/makefile
--- unrar.orig/makefile	2022-08-18 09:12:27.977466147 -0600
+++ unrar/makefile	2022-08-18 09:13:54.789536219 -0600
@@ -2,13 +2,13 @@
 # Makefile for UNIX - unrar
 
 # Linux using GCC
-CXX=c++
-CXXFLAGS=-O2 -Wno-logical-op-parentheses -Wno-switch -Wno-dangling-else
+CXX=$(DESTARCH)-tomatoware-linux-$(DESTARCHLIBC)$(EABI)-g++
+CXXFLAGS=-O2 -Wno-logical-op-parentheses -Wno-switch -Wno-dangling-else $(EXTRACFLAGS)
 LIBFLAGS=-fPIC
 DEFINES=-D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -DRAR_SMP
-STRIP=strip
-AR=ar
-LDFLAGS=-pthread
+STRIP=$(DESTARCH)-tomatoware-linux-$(DESTARCHLIBC)$(EABI)-strip
+AR=$(DESTARCH)-tomatoware-linux-$(DESTARCHLIBC)$(EABI)-ar
+LDFLAGS=$(_LDFLAGS)
 DESTDIR=/usr
 
 # Linux using LCC
