diff -Naur readline-8.1.2_orig/configure readline-8.1.2/configure
--- readline-8.1.2_orig/configure	2022-07-14 12:39:31.450261315 -0600
+++ readline-8.1.2/configure	2022-07-14 12:39:45.118132319 -0600
@@ -6852,10 +6852,10 @@
 $as_echo_n "checking configuration for building shared libraries... " >&6; }
         eval `TERMCAP_LIB=$TERMCAP_LIB ${CONFIG_SHELL-/bin/sh} ${srcdir}/support/shobj-conf -C "${CC}" -c ${host_cpu} -o ${host_os} -v ${host_vendor}`
 
-#	case "$SHLIB_LIBS" in
-#	*curses*|*termcap*|*termlib*)	;;
-#	*)			SHLIB_LIBS="$SHLIB_LIBS $TERMCAP_LIB" ;;
-#	esac
+	case "$SHLIB_LIBS" in
+	*curses*|*termcap*|*termlib*)	;;
+	*)			SHLIB_LIBS="$SHLIB_LIBS $TERMCAP_LIB" ;;
+	esac
 
 
 
