diff -Naur uClibc-ng-1.0.36_orig/ldso/ldso/dl-elf.c uClibc-ng-1.0.36/ldso/ldso/dl-elf.c
--- uClibc-ng-1.0.36_orig/ldso/ldso/dl-elf.c	2020-10-26 11:01:13.381237792 -0600
+++ uClibc-ng-1.0.36/ldso/ldso/dl-elf.c	2020-10-26 11:03:34.558725069 -0600
@@ -334,10 +334,9 @@
 	   This list must exactly match the list in uClibc/ldso/util/ldd.c */
 	_dl_if_debug_dprint("\tsearching full lib path list\n");
 	tpnt1 = search_for_named_library(libname, rflags,
-					UCLIBC_RUNTIME_PREFIX "lib:"
-					UCLIBC_RUNTIME_PREFIX "usr/lib"
+					"/mmc/lib:/mmc/usr/lib:/mmc/usr/local/lib"
 #ifndef __LDSO_CACHE_SUPPORT__
-					":" UCLIBC_RUNTIME_PREFIX "usr/X11R6/lib"
+					":" "/mmc/X11R6/lib"
 #endif
 					, rpnt, NULL);
 	if (tpnt1 != NULL)
