diff -Naur uClibc-ng-1.0.26_orig/ldso/ldso/dl-elf.c uClibc-ng-1.0.26/ldso/ldso/dl-elf.c
--- uClibc-ng-1.0.26_orig/ldso/ldso/dl-elf.c	2017-09-08 11:07:44.314799705 -0600
+++ uClibc-ng-1.0.26/ldso/ldso/dl-elf.c	2017-09-08 11:08:19.899208129 -0600
@@ -306,19 +306,6 @@
 		}
 	}
 #endif
-
-#ifdef LDSO_MULTILIB_DIR
-	/* If multilib directory is selected, search it before falling back to
-	   standard lib directories. */
-	_dl_if_debug_dprint("\tsearching multilib lib path list\n");
-	tpnt1 = search_for_named_library(libname, rflags,
-					UCLIBC_RUNTIME_PREFIX LDSO_MULTILIB_DIR ":"
-					UCLIBC_RUNTIME_PREFIX "usr" LDSO_MULTILIB_DIR,
-					rpnt, NULL);
-	if (tpnt1 != NULL)
-		return tpnt1;
-#endif
-
 #if defined SHARED && defined __LDSO_SEARCH_INTERP_PATH__
 	/* Look for libraries wherever the shared library loader
 	 * was installed */
@@ -327,7 +314,6 @@
 	if (tpnt1 != NULL)
 		return tpnt1;
 #endif
-
 	/* Lastly, search the standard list of paths for the library.
 	   This list must exactly match the list in uClibc/ldso/util/ldd.c */
 	_dl_if_debug_dprint("\tsearching full lib path list\n");
diff -Naur uClibc-ng-1.0.26_orig/ldso/ldso/Makefile.in uClibc-ng-1.0.26/ldso/ldso/Makefile.in
--- uClibc-ng-1.0.26_orig/ldso/ldso/Makefile.in	2017-09-08 11:07:44.314799705 -0600
+++ uClibc-ng-1.0.26/ldso/ldso/Makefile.in	2017-09-08 11:08:19.899208129 -0600
@@ -34,11 +34,6 @@
 
 CFLAGS-ldso.c := -DLDSO_ELFINTERP=\"$(TARGET_ARCH)/elfinterp.c\"
 
-# Search non-default multilib directories
-ifneq ($(MULTILIB_DIR),lib)
-CFLAGS-ldso.c	+= -DLDSO_MULTILIB_DIR=\"$(MULTILIB_DIR)\"
-endif
-
 ifeq ($(TARGET_ARCH),arc)
 CFLAGS-ldso.c += -mno-long-calls
 endif
