diff -Naur uClibc-ng-1.0.41_orig/include/stdlib.h uClibc-ng-1.0.41/include/stdlib.h
--- uClibc-ng-1.0.41_orig/include/stdlib.h	2022-05-20 14:19:08.378614551 -0600
+++ uClibc-ng-1.0.41/include/stdlib.h	2022-05-20 15:10:28.903811865 -0600
@@ -979,7 +979,7 @@
 # endif
 #endif
 
-#if 0 /* def __USE_BSD */
+#ifdef __USE_BSD
 /* Put the 1 minute, 5 minute and 15 minute load averages into the first
    NELEM elements of LOADAVG.  Return the number written (never more than
    three, but may be less than NELEM), or -1 if an error occurred.  */
diff -Naur uClibc-ng-1.0.41_orig/libc/misc/getloadavg/getloadavg.c uClibc-ng-1.0.41/libc/misc/getloadavg/getloadavg.c
--- uClibc-ng-1.0.41_orig/libc/misc/getloadavg/getloadavg.c	1969-12-31 17:00:00.000000000 -0700
+++ uClibc-ng-1.0.41/libc/misc/getloadavg/getloadavg.c	2022-05-20 15:04:48.486829670 -0600
@@ -0,0 +1,39 @@
+/*
+ * Copyright © 2005-2020 Rich Felker, et al.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining
+ * a copy of this software and associated documentation files (the
+ * "Software"), to deal in the Software without restriction, including
+ * without limitation the rights to use, copy, modify, merge, publish,
+ * distribute, sublicense, and/or sell copies of the Software, and to
+ * permit persons to whom the Software is furnished to do so, subject to
+ * the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be
+ * included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+ * IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+ * CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ *
+ * Imported from musl C library
+ */
+
+#define _GNU_SOURCE
+#include <stdlib.h>
+#include <sys/sysinfo.h>
+
+int getloadavg(double *a, int n)
+{
+	struct sysinfo si;
+	if (n <= 0) return n ? -1 : 0;
+	sysinfo(&si);
+	if (n > 3) n = 3;
+	for (int i=0; i<n; i++)
+		a[i] = 1.0/(1<<SI_LOAD_SHIFT) * si.loads[i];
+	return n;
+}
diff -Naur uClibc-ng-1.0.41_orig/libc/misc/getloadavg/Makefile uClibc-ng-1.0.41/libc/misc/getloadavg/Makefile
--- uClibc-ng-1.0.41_orig/libc/misc/getloadavg/Makefile	1969-12-31 17:00:00.000000000 -0700
+++ uClibc-ng-1.0.41/libc/misc/getloadavg/Makefile	2022-05-20 15:04:48.486829670 -0600
@@ -0,0 +1,13 @@
+# Makefile for uClibc
+#
+# Copyright (C) 2000-2005 Erik Andersen <andersen@uclibc.org>
+#
+# Licensed under the LGPL v2.1, see the file COPYING.LIB in this tarball.
+#
+
+top_srcdir=../../../
+top_builddir=../../../
+all: objs
+include $(top_builddir)Rules.mak
+include Makefile.in
+include $(top_srcdir)Makerules
diff -Naur uClibc-ng-1.0.41_orig/libc/misc/getloadavg/Makefile.in uClibc-ng-1.0.41/libc/misc/getloadavg/Makefile.in
--- uClibc-ng-1.0.41_orig/libc/misc/getloadavg/Makefile.in	1969-12-31 17:00:00.000000000 -0700
+++ uClibc-ng-1.0.41/libc/misc/getloadavg/Makefile.in	2022-05-20 15:04:48.486829670 -0600
@@ -0,0 +1,23 @@
+# Makefile for uClibc
+#
+# Copyright (C) 2000-2008 Erik Andersen <andersen@uclibc.org>
+#
+# Licensed under the LGPL v2.1, see the file COPYING.LIB in this tarball.
+#
+
+subdirs += libc/misc/getloadavg
+
+CSRC-y := getloadavg.c
+
+MISC_GETLOADAVG_DIR := $(top_srcdir)libc/misc/getloadavg
+MISC_GETLOADAVG_OUT := $(top_builddir)libc/misc/getloadavg
+
+MISC_GETLOADAVG_SRC := $(patsubst %.c,$(MISC_GETLOADAVG_DIR)/%.c,$(CSRC-y))
+MISC_GETLOADAVG_OBJ := $(patsubst %.c,$(MISC_GETLOADAVG_OUT)/%.o,$(CSRC-y))
+
+libc-y += $(MISC_GETLOADAVG_OBJ)
+
+objclean-y += CLEAN_libc/misc/getloadavg
+
+CLEAN_libc/misc/getloadavg:
+	$(do_rm) $(addprefix $(MISC_GETLOADAVG_OUT)/*., o os)
diff -Naur uClibc-ng-1.0.41_orig/libc/misc/Makefile.in uClibc-ng-1.0.41/libc/misc/Makefile.in
--- uClibc-ng-1.0.41_orig/libc/misc/Makefile.in	2022-05-20 14:19:08.358614521 -0600
+++ uClibc-ng-1.0.41/libc/misc/Makefile.in	2022-05-20 15:05:18.786538795 -0600
@@ -14,6 +14,7 @@
 include $(top_srcdir)libc/misc/fnmatch/Makefile.in
 include $(top_srcdir)libc/misc/ftw/Makefile.in
 include $(top_srcdir)libc/misc/fts/Makefile.in
+include $(top_srcdir)libc/misc/getloadavg/Makefile.in
 include $(top_srcdir)libc/misc/glob/Makefile.in
 include $(top_srcdir)libc/misc/internals/Makefile.in
 include $(top_srcdir)libc/misc/locale/Makefile.in
