diff -Naur buildroot-git.orig/package/musl/musl.mk buildroot-git/package/musl/musl.mk
--- buildroot-git.orig/package/musl/musl.mk	2022-09-07 10:23:40.014641411 -0600
+++ buildroot-git/package/musl/musl.mk	2022-09-07 10:24:46.958408396 -0600
@@ -26,6 +26,9 @@
 
 MUSL_INSTALL_STAGING = YES
 
+# Build position independent libc.a for -static-pie support.
+MUSL_EXTRA_CFLAGS += -fPIC
+
 # musl does not build with LTO, so explicitly disable it
 # when using a compiler that may have support for LTO
 ifeq ($(BR2_TOOLCHAIN_GCC_AT_LEAST_4_7),y)
