diff -Naur boost_1_70_0.orig/tools/build/src/tools/common.jam boost_1_70_0/tools/build/src/tools/common.jam
--- boost_1_70_0.orig/tools/build/src/tools/common.jam	2019-07-12 10:38:59.148864109 -0600
+++ boost_1_70_0/tools/build/src/tools/common.jam	2019-07-12 10:39:33.401165813 -0600
@@ -974,7 +974,7 @@
     }
 
     # From GCC 5, versioning changes and minor becomes patch
-    if $(tag) = gcc && [ numbers.less 4 $(version[1]) ]
+    if $(tag) = gcc #&& [ numbers.less 4 $(version[1]) ]
     {
         version = $(version[1]) ;
     }
