diff -Naur cmake-3.16.1.orig/Utilities/cmlibuv/src/unix/linux-core.c cmake-3.16.1/Utilities/cmlibuv/src/unix/linux-core.c
--- cmake-3.16.1.orig/Utilities/cmlibuv/src/unix/linux-core.c	2019-12-18 10:40:02.578693570 -0700
+++ cmake-3.16.1/Utilities/cmlibuv/src/unix/linux-core.c	2019-12-18 10:40:37.262973159 -0700
@@ -90,17 +90,17 @@
    * a.k.a. Lollipop. Since EPOLL_CLOEXEC is an alias for O_CLOEXEC on all
    * architectures, we just use that instead.
    */
-  fd = epoll_create1(O_CLOEXEC);
+//  fd = epoll_create1(O_CLOEXEC);
 
   /* epoll_create1() can fail either because it's not implemented (old kernel)
    * or because it doesn't understand the O_CLOEXEC flag.
    */
-  if (fd == -1 && (errno == ENOSYS || errno == EINVAL)) {
+//  if (fd == -1 && (errno == ENOSYS || errno == EINVAL)) {
     fd = epoll_create(256);
 
     if (fd != -1)
       uv__cloexec(fd, 1);
-  }
+//  }
 
   loop->backend_fd = fd;
   loop->inotify_fd = -1;
