diff -Naur Python-2.7.3/Parser/asdl_c.py Python-2.7.3_orig/Parser/asdl_c.py
--- Python-2.7.3/Parser/asdl_c.py	2012-04-09 17:07:35.000000000 -0600
+++ Python-2.7.3_orig/Parser/asdl_c.py	2015-02-09 07:38:55.584298909 -0700
@@ -1033,7 +1033,7 @@
             self.emit("case %s:" % t.name, 2)
             self.emit("Py_INCREF(%s_singleton);" % t.name, 3)
             self.emit("return %s_singleton;" % t.name, 3)
-        self.emit("default:" % name, 2)
+        self.emit("default:", 2)
         self.emit('/* should never happen, but just in case ... */', 3)
         code = "PyErr_Format(PyExc_SystemError, \"unknown %s found\");" % name
         self.emit(code, 3, reflow=False)
