From d0a2683120ec5a33163a14b0e1b39d208745968f Mon Sep 17 00:00:00 2001
From: Marco A L Barbosa <malbarbo@gmail.com>
Date: Tue, 22 Jun 2021 07:45:41 -0300
Subject: [PATCH] Fix strtonm run test when crosscompiling

---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index f6bdd3c73..30b4d3708 100644
--- a/configure.ac
+++ b/configure.ac
@@ -160,6 +160,7 @@ AC_RUN_IFELSE([AC_LANG_PROGRAM(
          [return (strtonum("0", 0, 1, NULL) == 0 ? 0 : 1);]
 	 )],
 	 [AC_DEFINE(HAVE_STRTONUM) AC_MSG_RESULT(yes)],
+	 [AC_LIBOBJ(strtonum) AC_MSG_RESULT(no)],
 	 [AC_LIBOBJ(strtonum) AC_MSG_RESULT(no)]
 )
 
