diff -Naur openssh-8.7p1.orig/Makefile.in openssh-8.7p1/Makefile.in
--- openssh-8.7p1.orig/Makefile.in	2021-09-22 10:37:36.011383269 -0600
+++ openssh-8.7p1/Makefile.in	2021-09-22 10:41:07.981298167 -0600
@@ -377,7 +377,7 @@
 	$(AUTORECONF)
 	-rm -rf autom4te.cache .depend.bak
 
-install: $(CONFIGFILES) $(MANPAGES) $(TARGETS) install-files install-sysconf host-key check-config
+install: $(CONFIGFILES) $(MANPAGES) $(TARGETS) install-files install-sysconf host-key
 install-nokeys: $(CONFIGFILES) $(MANPAGES) $(TARGETS) install-files install-sysconf
 install-nosysconf: $(CONFIGFILES) $(MANPAGES) $(TARGETS) install-files
 
